packages/text-editor/src/components/ReferenceInput.svelte

```diff
@@ -156,195 +157,133 @@
+        <div class="buttons-group xsmall-gap">
+          {#each actions as a}
+            <Button
+              icon={a.icon}
+              iconProps={{ size: buttonSize }}
+              kind="ghost"
+              showTooltip={{ label: a.label }}
+              size={buttonSize}
+              on:click={handler(a, (a, evt) => handleAction(a, evt))}
+            />
+            {#if a.order % 10 === 1}
+              <div class="buttons-divider" />
+            {/if}
+          {/each}
         </div>
-        {#if a.order % 10 === 1}
-          <div class="buttons-divider" />
+        {#if extraActions && extraActions.length > 0}
+          <div class="buttons-group {shrinkButtons ? 'xsmall-gap' : 'small-gap'}">
+            {#each extraActions as a}
+              <Button
+                disabled={a.disabled}
+                icon={a.icon}
+                iconProps={{ size: buttonSize }}
+                kind="ghost"
+                showTooltip={{ label: a.label }}
+                size={buttonSize}
+                on:click={handler(a, (a, evt) => {
+                  if (!a.disabled) {
+                    handleAction(a, evt)
+                  }
+                })}
+              />
+            {/each}
+          </div>
         {/if}
-      {/each}
+      {/if}
+
+      {#if showSend}
+        <Button
+          {loading}
+          disabled={(isEmpty && !haveAttachment) || loading}
+          icon={iconSend ?? Send}
+          iconProps={{ size: buttonSize }}
+          {kind}
+          size={buttonSize}
+          showTooltip={{
+            label: labelSend ?? textEditorPlugin.string.Send
+          }}
+          on:click={submit}
+        />
+      {/if}
     </div>
-    {#if extraActions && extraActions.length > 0}
-      <div class="buttons-group {shrinkButtons ? 'medium-gap' : 'large-gap'}">
-        {#each extraActions as a}
-          <!-- svelte-ignore a11y-click-events-have-key-events -->
-          <div
-            class="icon-button"
-            class:disabled={a.disabled}
-            use:tooltip={{ label: a.label }}
-            on:click={handler(a, (a, evt) => {
-              if (!a.disabled) {
-                handleAction(a, evt)
-              }
-            })}
-          >
-            <Icon icon={a.icon} size={'medium'} fill={a.fill} />
-          </div>
-        {/each}
-      </div>
-    {/if}
-  </div>
+  {/if}
 </div>
 
 <style lang="scss">
-  .icon-button {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 1.25rem;
-    height: 1.25rem;
-    color: var(--theme-darker-color);
-    cursor: pointer;
-
-    &:hover {
-      color: var(--theme-content-color);
-    }
-    &.disabled {
-      color: var(--theme-trans-color);
-      &:hover {
-        color: var(--theme-trans-color);
-        cursor: not-allowed;
-      }
-    }
-  }
   .ref-container {
     display: flex;
     flex-direction: column;
     min-height: 4.5rem;
+    border-width: 0.0625rem;
```

Comment:
Minor: use `border` short syntax for the first three styles?

Reply:
Updated